src/bionics.cpp

```diff
@@ -204,21 +204,18 @@ bionic_data const& bionic_info(std::string const &id)
     debugmsg("bad bionic id");
 
     static bionic_data const null_value {"bad bionic", false, false, 0, 0, 0, 0, 0,
-                                         "bad_bionic", false,
-                                         { num_bp , 1 } };
+                                         "bad_bionic", false };
     return null_value;
 }
 
 void bionics_install_failure( player *u, int difficulty, int success,
-                              std::map<body_part, int> affected_bodyparts );
+                              std::map<body_part, size_t> affected_bodyparts );
 
 bionic_data::bionic_data(std::string nname, bool ps, bool tog, int pac, int pad, int pot,
```

Comment:
Possibly move this to the header as it gives other compilation units a chance to inline it. Dependent upon how this is used (especially within loops) that could be a performance advantage.